diff --git a/node_modules/typed-ecstasy/src/core/Engine.ts b/node_modules/typed-ecstasy/src/core/Engine.ts
index ba1e1b9..251784b 100644
--- a/node_modules/typed-ecstasy/src/core/Engine.ts
+++ b/node_modules/typed-ecstasy/src/core/Engine.ts
@@ -68,7 +68,7 @@ export class Engine {
 	private entitiesByFamily: { [s: string]: Entity[] } = {};
 	private entityAddedSignals: { [s: string]: EntitySignal } = {};
 	private entityRemovedSignals: { [s: string]: EntitySignal } = {};
-	private entityFactory: EntityFactory;
+	private entityFactory!: EntityFactory;
 	private updating = false;
 	private notifying = false;
 	private nextEntityId = 1;
